src/FlipMove.js

```diff
@@ -69,7 +69,7 @@ class FlipMove extends Component {
     // can complete. Because we cannot mutate props, we make `state` the source
     // of truth.
     this.state = {
-      children: props.children.map(child => ({
+      children: Children.toArray(props.children).map(child => ({
```

Comment:
Hm.. The thing is, it shouldn't be possible to access `FlipMove` without going through `propConverter`. The export of `FlipMove.js` is `propConverter(FlipMove)`, so when someone imports it, they're actually importing the `FlipMovePropConverter` component.

Is there an internal mechanism with children I'm not familiar with?